ami/jobs/status.py

```
@@ -0,0 +1,424 @@
+    job.logger.warning(
+        f"Job status '{job.status}' doesn't match Celery task status '{celery_status}'. " f"Updating to match Celery."
+    )
+
+    old_status = job.status
+    job.update_status(celery_status, save=False)
+
+    # If Celery says it's in a final state but we thought it was running
+    if celery_status in JobState.final_states() and old_status in JobState.running_states():
+        job.finished_at = now
+        if celery_status in JobState.failed_states():
+            job.logger.error(f"Task failed in Celery")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)